.babelrc

```diff
@@ -1,4 +1,3 @@
 {
-  "stage": 0,
-  "loose": "all"
+  "presets": ["@babel/preset-env"]
```

Comment:
You can also use the shortname `@babel/env`: https://new.babeljs.io/docs/en/next/v7-migration.html#package-renames